ruby-sinatra/myapp.rb

```diff
@@ -25,32 +32,24 @@
   enable :sessions
 end
 
-configure :development do
-  set :database, { adapter: "postgresql", database: "minitwit_development" }
-  set :public_folder, "#{__dir__}/static"
-  ActiveRecord.verbose_query_logs = true
-  enable :sessions
-end
 
-configure :test do
+configure :development, :test do
   if DATABASE_URL
-    db = URI.parse(DATABASE_URL)
-    set :database, {
-      adapter: db.scheme,
-      host: db.host,
-      port: db.port,
-      database: db.path[1..],
-      user: db.user,
-      password: db.password,
-      encoding: "utf8",
-    }
+    set :database, DATABASE_URL
   else
-    set :database, { adapter: "postgresql", database: "minitwit_test" }
+    # Fall back to specific configurations for development and test environments
+    case settings.environment
+    when :development
```

Comment:
Maybe we can add the specified varibales here and comment out the host so we can run it locally. Depending on, if we're just using the image in the future
`{ adapter: "postgresql",
            database: "waect" ,
            host: "192.168.8.175",
            port: 5432,
            database: "waect",
            username: "user",
            password: "pass"
      }`